low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -95,12 +97,13 @@ class SSPLBundleGenerate(object):
                 # Capturing enclosure logs only when sspl state is active
                 sspl_state = sspl_state[1] if len(sspl_state)==2 else ''
                 if not sspl_state == 'active':
-                    print("SSPL is in 'degraded' mode, so enclosure data not collected as part of this node support bundle.")
+                    self._logger.info("SSPL is in 'degraded' mode,"
```

Comment:
Using logger is still accessible only. Why again reassignment of logger to `self._logger` ?

Reply:
Agree. Done the change